blog/apex.axdl

```diff
@@ -0,0 +1,69 @@
+import * from "@apexlang/core"
+import * from "@apexlang/rest"
+import * from "@apexlang/openapi"
+
+namespace "blogs.v1"
+@info(
+    title: "Simple blog app"
+    description: "Simple blog application created using NanoBus and IOtas."
+    version: "1.0.0"
+    termsOfService: "https://nanoblog.io/terms/"
+    contact: {
+      name: "API Support",
+      url: "https://nanoblog.io/support",
+      email: "api@nanoblog.io"
+    },
+    license: {
+      name: "Apache 2.0",
+      url: "https://www.apache.org/licenses/LICENSE-2.0"
+    }
+  )
+  @host("nanoblog.io")
+  @path("/v1")
+
+"Blog API"
+interface Blogs @service @path("/blogs") {
+  "Post a Tweet."
+  postBlog(
+    userId: string,
+    title: string,
+    body: string 
+  ): Blog @POST @nocode @path("/blog")
```

Comment:
Remove the @path here to fit REST resources semantics. URI should be POST /blogs.